pinot-core/src/main/java/org/apache/pinot/core/data/table/TableResizer.java

```diff
@@ -57,15 +57,18 @@
     int numAggregations = aggregationInfos.size();
     int numKeyColumns = numColumns - numAggregations;
 
+    int aggIdx = 0;
     Map<String, Integer> columnIndexMap = new HashMap<>();
+    Map<String, AggregationFunction> aggregationColumnToFunction = new HashMap<>();
     for (int i = 0; i < numColumns; i++) {
-      columnIndexMap.put(dataSchema.getColumnName(i), i);
-    }
-
-    Map<String, AggregationInfo> aggregationColumnToInfo = new HashMap<>();
-    for (AggregationInfo aggregationInfo : aggregationInfos) {
-      String aggregationColumn = AggregationFunctionUtils.getAggregationColumnName(aggregationInfo);
-      aggregationColumnToInfo.put(aggregationColumn, aggregationInfo);
+      String columnName = dataSchema.getColumnName(i).toLowerCase();
```

Comment:
This is not correct as `column` and `COLUMN` should be treated as two different columns (case sensitive)

Reply:
Changed this. I ended up making function name lower case in the AggregationFunction::getResultColumnName, because TransformExpressionTree converts function names to lower case when building the broker request. Hence order by only has all lower case function names